api/db/repositories/customerRepository.js

```diff
@@ -36,30 +36,30 @@ exports.deleteCustomer = async (customerId) => {
     }
 };
 
-exports.getAll = async (customers) => {
-    const requestedCustomer = await Customers.findAll(customers);
+exports.getAll = async () => {
+    const allCustomers = await Customers.findAll();
 
-    if(requestedCustomer === null || requestedCustomer === undefined) {
+    if (allCustomers === null || allCustomers === undefined) { 
         throw new Error('Customers table is empty. REPO');
     }
 
     try {
-        return Customers.findAll(customers);
-    } catch(error) {
+        return allCustomers;
```

Comment:
there is no point in puttin only `return allCustomers;` in a try catch block, because there's nothing that can go wrong when you're just returning value. Look at my previous comment to see how it could be done properly to avoid problems